src/cpp/session/SessionConsoleProcessApi.cpp

```diff
@@ -337,7 +339,7 @@ SEXP rs_terminalClear(SEXP idSEXP)
    // send the event to the client; if not connected, it will get the cleared
    // server-side buffer next time it connects.
    json::Object eventData;
-   eventData["id"] = terminalId;
+   eventData["id"] = proc->getCaption();
```

Comment:
Is it worth making the change here to read `eventData["caption"] = proc->getCaption()`? Or is the notion of `id` on the client side still linked to the caption value? (I see we enforce that captions are unique in `terminalCreate` which I think is the right choice to avoid confusion)

Reply:
The client-side still takes a caption. To minimize the size of the change for 1.1 I left it that way, but will clean it up in 1.2. The user can't rename the caption while this API is in flight, so won't suffer the problem I'm fixing overall.

Reply:
Got it!